WcaOnRails/app/jobs/timed_application_job.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+class TimedApplicationJob < ApplicationJob
```

Comment:
Aha, it's part of the Rails 5 framework now!

Ideally there would be comment or test ensuring that it's not removed.